src/Pdp/PublicSuffixListManager.php

```diff
@@ -20,6 +20,13 @@ class PublicSuffixListManager
 {
     const PDP_PSL_TEXT_FILE = 'public-suffix-list.txt';
     const PDP_PSL_PHP_FILE = 'public-suffix-list.php';
+
+    const ICANN_SECTION = 'ICANN';
```

Comment:
Okay, I'll use renamed the constant using the `_DOMAINS` prefix and add the missing constant 👍